### ISSUE TITLE
Custom text in links is removed

### ISSUE THREAD

**Issue by sojusnik** [peter88213/zim2obsidian#16]
**Describe the bug**
After applying all recommended steps of this script (regardless of whether subst_indent.py and indent_md.py are used), the custom text in links is removed. Instead of the custom text, only the filename of the note is shown, always ending on .md., or if it's linked to a note within a directory, then the full path to the note is shown. The custom link text is displayed properly after the conversion with Zim's built-in markdown export function though.

See the following examples as source text in Zim:
```
See this link leading to a [[category:note|subnote.]]

Example of a link to a note at the end of a sentence with ° as custom text.[[category:note:same name note|°]]
Example of a link to a URL at the end of a sentence with ° as custom text.[[https://github.com/peter88213/zim2obsidian/|°]]
```

Then there's another issue that might be connected to the first one. If within one sentence several links are present and the last one is a link to a URL, then the whole text including the links before the URL is completely removed after applying your script, although it's displayed fine after Zim's built-in markdown export, see:
```
See this link leading to a [[category:note|subnote]] and this one to a [[same level note]].

See this link leading to a [[category:note|subnote]] and this one to a [[same level note]] and this link pointing to a [[https://github.com/peter88213/zim2obsidian/|URL]] as examples.
```

**To reproduce**
1. Copy the code examples into Zim
2. Create the linked notes there
3. Apply this script (with or without subst_indent.py and indent_md.py)
4. Open the note with the examples in Obsidian

**Expected behavior**
The custom text of the link should be preserved.

Also, the linked note path should be displayed without the .md ending, as Obsidian encourages using the [Wikilink format](https://help.obsidian.md/Linking+notes+and+files/Internal+links#Supported+formats+for+internal+links).

**Application/script**
- Version: 0.10.3

**Execution environment**
- Python: 3.11.2
- OS: Ubuntu 23.04

**Comment by peter88213** [comment]
If you change line 70, is it better?
Try his:
````
REFORMAT_LINKS = False
```

**Comment by peter88213** [comment]
I released v0.10.5 now, with the wikilinks conversion reworked yet disabled. 
You can try out the new conversion by changing line 72 to
```
REFORMAT_LINKS = True
```

**Comment by sojusnik** [comment]
Your tweaks did help a little, thanks!

For the sake of clarity, I've created two different issues for [Markdown](https://github.com/peter88213/zim2obsidian/issues/17) and [Wikilinks](https://github.com/peter88213/zim2obsidian/issues/18) style linking.
